### DISCUSSION TITLE
how to access then settings of a bottle once you have set it up

### DISCUSSION THREAD

**Discussion by aidansstuff** [The-Wineskin-Project/WineskinServer#390] [Q&A]
How to access the settings of a bottle once you have set it up. I cant get back into the menu after I set the bottle up.

**Comment by Igor-Ruk** [comment]
@aidansstuff, right-click on the created ___.app and select "Show Package Contents".
Once inside, open the "Wineskin.app".
<img width="365" alt="image" src="https://github.com/Gcenx/WineskinServer/assets/125645352/c0a030ee-ff66-44be-8a40-e6860c836eec">
